### PR TITLE
Document using `wire:model` in view fields

### PR DESCRIPTION
**Pull request by Shelob9**
I wasn't sure how to use `wire:model` today in a view field and wasn't sure how that would work. I wanted to document how this works for others.

### CODE REVIEW

File: packages/forms/docs/03-fields.md [hunk 1]
```diff
@@ -1916,6 +1918,23 @@ The `$getStatePath()` closure may be used by the view to retrieve the Livewire p
 </x-forms::field-wrapper>
 ```
 
+You can also bind with [`wire:model`](https://laravel-livewire.com/docs/properties#data-binding) :
```
Comment (zepfietje) [inline packages/forms/docs/03-fields.md:1921]:
```suggestion
You may bind the value to a Livewire property using [`wire:model`](https://laravel-livewire.com/docs/properties#data-binding):
```

File: packages/forms/docs/03-fields.md [hunk 2]
```diff
@@ -1899,6 +1899,8 @@ Inside your view, you may interact with the state of the form component using Li
 
 The `$getStatePath()` closure may be used by the view to retrieve the Livewire property path of the field. You could use this to [`wire:model`](https://laravel-livewire.com/docs/properties#data-binding) a value, or [`$wire.entangle`](https://laravel-livewire.com/docs/alpine-js) it with Alpine.js:
```
Comment (zepfietje) [inline packages/forms/docs/03-fields.md:1900]:
```suggestion
The `$getStatePath()` closure may be used by the view to retrieve the Livewire property path of the field. You could use this to [`wire:model`](https://laravel-livewire.com/docs/properties#data-binding) a value, or [`$wire.entangle`](https://laravel-livewire.com/docs/alpine-js) it with Alpine.js.
```

**Review by zepfietje — CHANGES_REQUESTED** [review]
Could you also move the `wire:model` example above the `$wire.entangle` example?

**Comment by Shelob9** [comment]
@zepfietje I accepted your suggestions.

**Comment by danharrin** [comment]
The entangle example should be first, that's much more common.

**Comment by zepfietje** [comment]
Alright, I thought it made sense to have them in the order they're mentioned in the sentence above.

**Comment by danharrin** [comment]
Ah I didn't see that, the sentence needs changing then

**Pull request merged**